<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsWidget</class>
 <widget class="QWidget" name="ResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnShowReport">
       <property name="text">
        <string>Показать отчёт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChowCharts">
       <property name="text">
        <string>Показать графики</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Экспорт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExportHTML">
       <property name="text">
        <string>Экспорт в HTML</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableView" name="tblResults">
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Гипотеза</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="radHypSimple">
            <property name="text">
             <string>Простая</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radHypLemeshko">
            <property name="text">
             <string>Сложная по Лемешко</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radHypShitikov">
            <property name="text">
             <string>Сложная по Шитикову</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Количество итераций:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="IntNumberEdit" name="txtNumIters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbxRound">
            <property name="text">
             <string>Округлять выборки</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="lblNumDigits">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Количество знаков: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="sbxNumDigits">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>0</number>
              </property>
              <property name="maximum">
               <number>5</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Расчёт вероятности</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="cbxItervalEstimation">
            <property name="text">
             <string>Вкл. интервальные оценки</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblToleranceProb">
            <property name="text">
             <string>Доверительная вероятность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="DoubleNumberEdit" name="txtToleranceProb">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblInrangeNumIters">
            <property name="text">
             <string>Количество итераций</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="IntNumberEdit" name="txtInrangeNumIters">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Настройки расчёта</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество потоков:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbxThreadsCount">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QProgressBar" name="progress">
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCalculate">
       <property name="text">
        <string>Вычислить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>extra_widgets/numberedit.h</header>
  </customwidget>
  <customwidget>
   <class>IntNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>extra_widgets/numberedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnCalculate</sender>
   <signal>clicked()</signal>
   <receiver>ResultsWidget</receiver>
   <slot>slotCalculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>666</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>749</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked()</signal>
   <receiver>ResultsWidget</receiver>
   <slot>slotExport2Math()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>541</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnChowCharts</sender>
   <signal>clicked()</signal>
   <receiver>ResultsWidget</receiver>
   <slot>slotShowChart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCalculate()</slot>
  <slot>slotExport2Math()</slot>
  <slot>slotShowChart()</slot>
 </slots>
</ui>
